app/controllers/users_controller.rb

```diff
@@ -44,12 +57,23 @@ def add_contact
     redirect_to root_path
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def search
-    @users = User.search(params[:search]).where.not(id: current_user.id)
-    @users_to_add = @users.reject do |user|
-      current_user.sent_contact_request?(user)
+    use_wildcards = true
```

Comment:
if you are just turning that to `true` to give that to the method, why are you not putting `true` into the method call?

Reply:
Because it is currently not necessary to switch wildcards off, I would completely omit this variable and always use wildcards.

Reply:
I replaced the search logic with the one from Scrumbledore's Army. So we are not using this variable anymore.